lib/database/sthDatabase.js

```diff
@@ -90,7 +90,7 @@ function getJSONCSVOptions(attrName) {
 function connect(params, callback) {
     connectionURL =
         // prettier-ignore
-        'mongodb://' + params.authentication + '@' + params.dbURI + '/' + params.database +
+        'mongodb://' + params.authentication + params.dbURI + '/' + params.database +
```

Comment:
According with: https://docs.mongodb.com/manual/reference/connection-string/ when auth is provided '@' should be used

Reply:
A fix to allow that was done in PR https://github.com/telefonicaid/fiware-sth-comet/pull/564